### PR TITLE
Setup SQLAlchemy with SQLite database for Flask app

### PR DESCRIPTION
**Pull request by paulfiesel**
- Added SQLAlchemy to manage SQLite database
- Configured database URL to use an instance folder for app.db
- Added create_app() function to initialize the Flask application with SQLAlchemy

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+db = SQLAlchemy()
 
-@app.route('/healthz')
-def healthz():
-    return '', 200
+def create_app():
+    app = Flask(__name__)
+
+    # Configure the SQLite database path
+    instance_path = os.path.expanduser('~/td2stats_instance')
```
Comment (hansenreid) [inline backend/app.py:12]: Normally you should place the database file in the directory of the repo. That way it will work on anyone's machine.

**Review by hansenreid — CHANGES_REQUESTED** [review]
Looks good, just need to change where the db is saved.

File: backend/app.py (hunk 1 above)
Comment (paulfiesel) [reply backend/app.py:12]: You're talking specifically about my instance folder contents, or the create_db.py file?

**Comment by paulfiesel** [comment]
part of #19

File: backend/app.py (hunk 1 above)
Comment (hansenreid) [reply backend/app.py:12]: The app.db file. And then you don't need this folder.

Comment (paulfiesel) [reply backend/app.py:12]: Screenshot 2024-10-30 222359

**Pull request closed without merging**